string  sensorName
float32 indicator
string  activatorName
